les/distributor.go

```diff
@@ -140,6 +146,7 @@ func (d *requestDistributor) loop() {
 					send := req.request(peer)
 					if send != nil {
 						peer.queueSend(send)
+						requestSendDelay.UpdateSince(req.enterQueue)
```

Comment:
I use mclock (monotonic absolute time) throughout the code to avoid glitches caused by system time adjustments. Maybe not a big issue but why not use the same clock here? It could save us some time scratching our heads later when we see some weird outlier data points in these statistics :) Also, the monotonic clock is really quick because it avoids conversion to system time which is totally unnecessary if you are interested in relative durations.
This applies to other uses of UpdateSince too.

Reply:
Fixed

Reply:
Please do the same thing for requestRTT, blockProcessingTimer and connectionTimer too.